packages/internal-scripts/src/release-image.ts

```diff
@@ -0,0 +1,159 @@
+/**
+ * @license
+ * Copyright (c) 2019 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
```

Comment:
I know this was carried over from the PolymerLabs org repo but when it lives here perhaps it's fine and appropriate to add the usual license header?

Reply:
Done! Changed to the shorter license - maintaining the correct year.